pages/common/multitail.md

```diff
@@ -0,0 +1,19 @@
+# multitail
+
+> Monitor last part of multiple files in multiple windows in a terminal.
```

Comment:
What does "Monitor" mean here ? Does it continuously read like `tail -f` ? If so, we should clarify that behavior.

Reply:
Rephrasing … what do you think about the following?

```
Keep showing continuously added data of multiple files in multiple windows.
```

Reply:
Keeping in line with my previous comment - `Follow output of multiple files in separate windows.`